airgun/entities/host.py

```diff
@@ -70,14 +70,18 @@ def update(self, entity_name, values):
         view.flash.assert_no_error()
         view.flash.dismiss()
 
-    def delete(self, entity_name):
+    def delete(self, entity_name, get_alert_text=False):
```

Comment:
To support more tests We can add a confirm=True kwargs :
in case of True  we handle the alert with Ok, otherwise we push cancel button,
and always return the alert text.